3_javascript/javascript1/week2/script.js

```diff
@@ -0,0 +1,129 @@
+/** Smart-ease - Goes Global! **/
+
+function getFullname(Gender, firstName, surName, useFormalName) {
+  let fullName = "";
+
+  if (Gender === "female") {
+    if (useFormalName === true) {
```

Comment:
The if (useFormalName === true) condition can be simplified because useFormalName is already a boolean value. So you can simply write 
```
if(useFormalName) {
...
}
```